components/count-badge/count-badge-mixin.js

```diff
@@ -178,8 +182,8 @@ export const CountBadgeMixin = superclass => class extends LocalizeCoreElement(R
 	renderTooltips(badgeId) {
 		return html`
 			${this.hasTooltip  ?
-		html`<d2l-tooltip aria-live="${this.announceChanges ? 'polite' : 'off'}" for="${badgeId}" for-type="label">${this.text}</d2l-tooltip>`
-		: html`<span id="${this._textId}" aria-live="${this.announceChanges ? 'polite' : 'off'}" class="d2l-offscreen">"${this.text}"</span>`}
+		html`<d2l-tooltip aria-atomic="true" aria-live="${this.announceChanges ? 'polite' : 'off'}" for="${badgeId}" for-type="label">${this.text}</d2l-tooltip>`
```

Comment:
Fun browser tidbit, chrome defaults to aria-atomic=true, while firefox does not. Firefox was saying some really funky things when announcing the changes (like only saying the last digit that changed?). Aria-atomic makes sure the WHOLE message is repeated when one portion changes.

Reply:
TIL!